cos-fleetshard-operator-camel-it/pom.xml

```diff
@@ -50,6 +50,8 @@
                         <java.util.logging.manager>org.jboss.logmanager.LogManager</java.util.logging.manager>
                         <maven.home>${maven.home}</maven.home>
                     </systemPropertyVariables>
+                    <forkCount>1</forkCount>
```

Comment:
I don't think this has any effect as it is the default, does it? I think it even yields a notification that this has no effect and considered a more or less invalid config?

Reply:
I have not noticed any warning but I may have missed it. 

I usually explicitly set values to avoid inherited values but I could remove it.

Reply:
I just remember some issue with that value recently with the DBZ UI. Maybe @Naros  remembers when we hit that?

Reply:
maven does not seem to complain

```
[INFO] --- maven-surefire-plugin:3.0.0-M6:test (default-test) @ cos-fleetshard-sync-it ---
[INFO] Using auto detected provider org.apache.maven.surefire.junitplatform.JUnitPlatformProvider
[INFO] 
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
```

Reply:
@rk3rn3r @Naros is shit a blocking issue ? or we can address it in a follow up PR if needed ?

The reason it has been added here is to be sure tests are executed in a new jvm since otherwise this could lead to OOM exception on GitHub Actions.